docs/en/sql-reference/aggregate-functions/parametric-functions.md

```diff
@@ -506,3 +506,254 @@ Solution: Write in the GROUP BY query SearchPhrase HAVING uniqUpTo(4)(UserID) >=
 ## sumMapFiltered(keys_to_keep)(keys, values) {#summapfilteredkeys-to-keepkeys-values}
 
 Same behavior as [sumMap](../../sql-reference/aggregate-functions/reference/summap.md#agg_functions-summap) except that an array of keys is passed as a parameter. This can be especially useful when working with a high cardinality of keys.
+
+## sequenceNextNode {#sequenceNextNode}
+
+Returns a value of next event that matched an event chain.
+
+**Syntax**
+
+``` sql
+sequenceNextNode(direction, base)(timestamp, base_condition, event_column, event1, event2, event3, ...)
+```
+
+**Parameters**
+-   `direction` - Used to navigate to directions.
+    - forward : Moving forward
```

Comment:
`forward` with `last_match` and `backward` with `first_match` are useful.
Because `the first matched event` and `the last matched event` will be in the middle of events so that a user want to navigate in both of directions.

`forward` with `tail` and `backward` with `head` is useless.